Doc/library/argparse.rst

```diff
@@ -636,6 +636,25 @@ be positional::
    usage: PROG [-h] [-f FOO] bar
    PROG: error: the following arguments are required: bar
 
+By default, argparse automatically handles the internal naming and
+display names of arguments, simplifying the process without requiring
+additional configuration.
+As such, you do not need to specify the dest_ and metavar_ parameters.
+The dest_ parameter defaults to the argument name with underscores ``_``
+replacing hyphens ``-`` . The metavar_ parameter defaults to the
+upper-cased name. For example::
```

Comment:
```suggestion
By default, argparse automatically handles the internal naming and
display names of arguments, simplifying the process without requiring
additional configuration. As such, you do not need to specify the dest_ 
and metavar_ parameters. The dest_ parameter defaults to the argument 
name with underscores ``_`` replacing hyphens ``-`` . The metavar_ 
parameter defaults to the upper-cased name. For example::
```

Reply:
Just removing the newline as I think that this can be one short paragraph.

Reply:
This does not affect the result. Sphinx reformat text.

I prefer to start new sentence from a new line. This will help to keep the text more readable in future without reformatting the whole paragraph. Adding or removing will only affect one sentence.

I don't know if this is already in the style guide, but I try to follow this rule.